src/air/tags/models/svg.py

```diff
@@ -15,16 +15,13 @@ def name(self) -> str:
         return self._name[0].lower() + self._name[1:]
 
 
-def svg_locals_cleanup(local_data):
+def svg_locals_cleanup(
+    data: dict[str, Any],
+    _skip: frozenset[str] = frozenset({"self", "children", "kwargs"}),
```

Comment:
IMO this would be better served with a `tuple` than a `set`. The former is significantly faster. We'll handle optimization at another time.